mitodl/bootcamp-ecommerce
# Skeleton UI

#### What are the relevant tickets?
Fixes #19

#### What's this PR do?
- Adds backend logic to populate profile
- Adds frontend logic for mock UI. The payment button will submit an API request but no API exists currently.

#### How should this be manually tested?
Login, then type a number in the textbox. Open the network tab. Click 'Pay' and verify that the total you entered was sent to `/api/v0/payment/`.
